README.md

```diff
@@ -204,26 +209,43 @@ public IEnumerator VisualTestStub_Generate()
 
 ### Making a manual build
 
-1. Build unity WASM with its name as `unity` into the folder `/static`. **It's very important that the folder/build name is `unity`**.
-2. Checkout the file named `static/unity/Build/DCLUnityLoader.js`. Unity deletes anything on this folder as part of the build process and we need that.
+#### Option 1
+1. Run `make watch` in a terminal at `kernel/static`. Wait for it to finish and leave it watching for changes. **This step must be done before replacing the build files or they may be replaced again with the latest published build files when the `make watch` process runs**
+2. Build unity WASM with its name as `unity` into the folder `kernel/static`. **It's very important that the folder/build name is `unity`**.
+3. In another terminal checkout the deletion of the file named `kernel/static/unity/Build/DCLUnityLoader.js`. Unity deletes anything on this folder as part of the build process and we need that file.
 
 ```
-git checkout -- static/unity/Build/DCLUnityLoader.js
+git checkout -- kernel/static/unity/Build/DCLUnityLoader.js
 ```
 
-4. Checkout the modifications in the `static/unity/Build/unity.json` file. Unity deletes our custom changes as part of the build process.
+4. Checkout the modifications in the `kernel/static/unity/Build/unity.json` file. Unity deletes our custom changes as part of the build process.
 
 ```
-git checkout -- static/unity/Build/unity.json
+git checkout -- kernel/static/unity/Build/unity.json
 ```
 
-5. Run `make watch`.
 6. Testing how your new build performs:
 
 - Open **[http://localhost:8080/?DEBUG_MODE&LOCAL_COMMS&position=-100,100](http://localhost:8080/?DEBUG_MODE&LOCAL_COMMS&position=-100,100)** to go to an area with a high density of test parcels.
 - Open **[http://localhost:8080/?DEBUG_MODE&LOCAL_COMMS&ENV=org&position=10,0](http://localhost:8080/?DEBUG_MODE&LOCAL_COMMS&ENV=org&position=10,0)** to open an area with real-life deployments (but without communicating with other users).
 - Open **[http://localhost:8080/?ENV=org&position=0,90](http://localhost:8080/?ENV=org&position=0,90)** to open the explorer near the Decentraland Museum
 
+#### Option 2
+Alternatively you can build into any other folder in your disk and then just move the 3 ".unityweb" files that will end up inside `[ChosenDestination]/unity/Build/` into `kernel/static/unity/Build/`. Just remember that if you run the `make watch` after this, your new files may be replaced by the latest published build files during the `make watch` process.
```

Comment:
Can we make this Option 1? cause current Option 1 is over-complicated

Reply:
totally

Reply:
Done